docs/wiki/Getting started - Scenario 2 Onboard module library and CI environment.md

```diff
@@ -208,6 +208,8 @@ The primary pipeline variable file ([`global.variables.yml`](https://github.com/
 
 > **NOTE:** If you plan to use the private container registry for Bicep, make sure to update its value in `bicepRegistryName`, as it must be globally unique.
 
+> **NOTE:** If you are uncertain which publishing option to choose, [this section](./The%20CI%20environment%20-%20Publishing#considerations) may be of help.
```

Comment:
I'm wondering if we need to reference these guidelines also outside of the CI environment. Afterall users (or better Solution Developers) may want to reference published modules for solutions even when not leveraging the CI environment. What's your take on that?

Reply:
Sounds legit. Would you have suggestions where?

I assume
- Solution Creation
- Context?

Reply:
Agree. Maybe a few lines with a quick summary in the context + link. Then if you agree I'd probably move the content to the solution creation (or a new subpage?) And link the guidelines from the ci environment publishing to the new section.

Reply:
Whatever you prefer. Just added it to the publishing as I felt 'talking about publishing locations in the publishing section' makes sense :D . Anyways, my main concern is that we reference it somewhere in the initial setup (which is already the case) as that where it was pointed out that it's missing.